### PR TITLE
Add prompts for schema_guided_dstc8 response generation

### PR DESCRIPTION
**Pull request by kasnerz**
Added 5 prompts for chatbot response generation

### CODE REVIEW

**Comment by stephenbach** [comment]
Thanks! Looks good! Just a couple of quick questions and thoughts:
- Should these be marked as "original task?"
- There are some new lines at the end of the targets that are added in some the prompt templates. These won't matter for methods that don't tokenize newlines separately, but probably best just to remove them. I assume you don't want to score methods based on that?

**Comment by kasnerz** [comment]
- No, I do not think the prompts should be marked as "original task". The dataset was developed with dialogue state tracking for the [DSTC8 challenge](https://arxiv.org/pdf/2002.01359.pdf) in mind, which is quite different from the (stateless) response generation.
- The newlines are not intentional, it is probably just an artifact from the web interface. I tried to remove them in the YAML file, please check if I did it correctly. I also noticed the same pattern is present in other prompts (e.g. MetaWOZ which I was using as an inspiration).

**Comment by kasnerz** [comment]
@stephenbach Done

File: promptsource/app.py [hunk 1]
```diff
@@ -640,4 +640,4 @@ def show_text(t, width=WIDTH, with_markdown=False):
 
 
 if __name__ == "__main__":
-    run_app()
+    run_app()
```
Comment (stephenbach) [inline promptsource/app.py:643]:
```suggestion
    run_app()

```

**Pull request merged**